sources/packages/backend/apps/api/src/services/education-program/education-program.service.ts

```diff
@@ -634,4 +666,27 @@ export class EducationProgramService extends RecordDataModelService<EducationPro
       },
     });
   }
+  /**
+   * Check if the given institution has already a program with the given SABC code.
+   * @param institutionId id of the institution to have the programs retrieved.
+   * @param sabcCode SABC code.
+   * @returns the id of the first record found.
```

Comment:
Please adjust the return comment to the boolean value returned.

Reply:
ok. Done. Thanks!